SECURITY.md

```diff
@@ -0,0 +1,47 @@
+# Security Policy
+
+## Supported Versions
+
+| Version   | Supported          |
+| --------- | ------------------ |
+| 1.0.x     | :white_check_mark: |
+| 0.38.5    | :white_check_mark: |
+| <= 0.38.4 | :x:                |
+
+## Reporting Security Issues
+
+⚠ **Please do not report security vulnerabilities through public GitHub issues.** ⚠
+
+Instead, either encrypted through Keybase chat
+  - [keybase.io/cakebuild](https://keybase.io/cakebuild)
+
+Or report them to using NuGet.org *Contact owners*
+  - [Cake](https://www.nuget.org/packages/Cake)
+  - [Cake.Tool](https://www.nuget.org/packages/Cake.Tool)
+  - [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting)
+  - [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR)
+  - [Cake.Core](https://www.nuget.org/packages/Cake.Core)
+  - [Cake.Common](https://www.nuget.org/packages/Cake.Common)
+  - [Cake.Testing](https://www.nuget.org/packages/Cake.Testing)
+  - [Cake.NuGet](https://www.nuget.org/packages/Cake.NuGet)
+  - [Cake.Cli](https://www.nuget.org/packages/Cake.Cli)
+
+If you prefer to submit without logging in, send email to [security@cakebuild.net](mailto:security@cakebuild.net).
```

Comment:
Tiny suggestion: "... send **an** email to ..."

Reply:
Sure I'll adjust.